server/app/controllers/LoginController.java

```diff
@@ -47,12 +47,13 @@ public class LoginController extends Controller {
   public LoginController(
       @AdminAuthClient @Nullable IndirectClient adminClient,
       @ApplicantAuthClient @Nullable IndirectClient applicantClient,
+      CiviFormHttpActionAdapter civiFormHttpActionAdapter,
       SessionStore sessionStore,
       Config config) {
     this.adminClient = adminClient;
     this.applicantClient = applicantClient;
     this.sessionStore = Preconditions.checkNotNull(sessionStore);
-    this.httpActionAdapter = PlayHttpActionAdapter.INSTANCE;
+    this.httpActionAdapter = civiFormHttpActionAdapter;
```

Comment:
What is this change doing?

Reply:
This just aligns the login error handler with the custom one Bion made for the API.  CiviFormHttpActionAdapter inherits from PlayHttpActionAdapter, and currently only overrides responses for API requests.  I mainly did this so that we don't have 2 different paths for errors.